Grids AI Column: Update Hiding Info (#9058)
DevExpress/devextreme-documentation: api-reference/10 UI Components/GridBase/1 Configuration/columnHidingEnabled.md

[note]

- When this property is enabled and [columnResizingMode](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/columnResizingMode.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#columnResizingMode') is set to *"nextColumn"* (default), enable [columnAutoWidth](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/columnAutoWidth.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#columnAutoWidth').

> - {WidgetName} does not hide command columns or [AI columns](/Documentation/Guide/UI_Components/DataGrid/Columns/Column_Types/AI_Columns/) into the adaptive detail view.

Reviewer: ```suggestion
- {WidgetName} does not move command columns or [AI columns](/Documentation/Guide/UI_Components/DataGrid/Columns/Column_Types/AI_Columns/) to the adaptive detail view.
```
